tests/integration/service/test_integration_files_service.py

```diff
@@ -212,7 +212,7 @@ async def test_upload_texts(self, integration_config: CommonConfig, workspace_na
                 DeepsetCloudFile("file4", "file4.txt", {"which": 4}),
                 DeepsetCloudFile("file5", "file5.txt", {"which": 5}),
             ]
-            result = await file_service.upload_texts(
+            result = await file_service.upload_in_memory(
```

Comment:
so basically here an other test for the other memory file class

Reply:
added a pdf file in the above list of files using the DeepsetCloudFileBytes class